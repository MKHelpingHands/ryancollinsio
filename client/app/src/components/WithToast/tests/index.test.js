import WithToast from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<WithToast />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <WithToast />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
