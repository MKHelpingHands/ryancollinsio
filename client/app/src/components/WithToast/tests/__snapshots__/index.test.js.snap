exports[`<WithToast /> should render with default props 1`] = `<div />`;
